shared/actions/config.js

```diff
@@ -225,12 +225,10 @@ function _loadAvatarHelper(action: AvatarHelperAction) {
 }
 
 function _loadAvatarHelperError(error: Error, action: AvatarHelperAction) {
-  // Ignore api errors
-  if (error.code === RPCTypes.constantsStatusCode.scapinetworkerror) {
-    throw error // TEMP
-  } else {
+  if (error.code === RPCTypes.constantsStatusCode.scinputerror) {
     throw error
   }
+  // ignore all other errors
```

Comment:
worth calling `log.warn` or `log.info`?

Reply:
i think in practice it'll just be noise. the avatars load all the time and if you're on mobile it'll just give you a ton of variants of being offline (dns/retry/availability etc)